doc/api/cli.md

```diff
@@ -194,6 +194,9 @@ process.
 <!-- YAML
 added: v20.0.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/58579
+    description: Entrypoint of your application are allowed to read implicitly.
```

Comment:
```suggestion
    description: Entrypoints of your application are allowed to be read implicitly.
```

Otherwise it sounds like the entrypoint would be allowed to read from any file